tools/generate_docs.sh

```diff
@@ -27,9 +27,9 @@ protoc --plugin=protoc-gen-doc="/usr/local/bin/protoc-gen-doc" --doc_out="$targe
 echo "Generate Markdown from ./api/proto/ankaios.proto done."
 
 if [[ "$1" = serve ]]; then
-    mkdocs serve -f "$target_dir/mkdocs.yml"
+    mkdocs serve --config-file "$target_dir/mkdocs.yml"
```

Comment:
I tried to serve the documentation locally to check how it works and I'm getting warnings:
```
WARNING -  [10:06:55] "GET /versions.json HTTP/1.1" code 404
```
I searched a bit and found this issue here: https://github.com/squidfunk/mkdocs-material/issues/3164

Should we use `mike` here?

```suggestion
    mike serve --config-file "$target_dir/mkdocs.yml"
```

Reply:
It's not that easy. `mkdocs serve` and `mike serve` are quite different:

* `mkdocs serve`: Build the documentation and watch for changes and serve the result.
* `mike serve`: Uses an already built documentation which has been committed to the `gh-pages` branch. So it is neither building nor watching.

From my point of view the warning is not that urgent and could be ignored. For preview we do not need the features of mike like switching between different versions.

Reply:
Alright.